synth_machine/executors/fake.py

```diff
@@ -0,0 +1,120 @@
+from typing import AsyncGenerator, Optional
+
+from synth_machine.executors.base import BaseExecutor, singleton
+from synth_machine.machine_config import (
+    ModelConfig,
+    calculate_input_tokens,
+)
+import random
+import asyncio
+
+
+@singleton
+class FakeExecutor(BaseExecutor):
+    def __init__(self) -> None:
+        self.word_catalog = [
```

Comment:
is this really needed? create a fixture if you really want this. This should be basically empty.